java-squid/src/main/java/org/sonar/java/model/declaration/VariableTreeImpl.java

```diff
@@ -47,6 +49,8 @@
   private IdentifierTree simpleName;
   @Nullable
   private ExpressionTree initializer;
+  @Nullable
+  private SyntaxToken endToken;
 
   // FIXME(Godin): never should be null, i.e. should have default value
```

Comment:
![MAJOR](https://raw.githubusercontent.com/henryju/image-hosting/master/major.png)Take the required action to fix the issue indicated by this comment.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)